CRVResponse/fcl/prolog_v11.fcl

```diff
@@ -127,7 +127,7 @@ BEGIN_PROLOG
       module_type                  : CrvWaveformsGenerator
       crvSiPMChargesModuleLabel    : "CrvSiPMCharges"
       digitizationStart            :  400.0    //400ns
-      digitizationEnd              : 1750.0    //1750ns
+      digitizationEnd              : 1695.0    //1695ns
```

Comment:
Please consolidate all these numbers to a single fcl block in prolog to avoid replication.

Reply:
I like you have units in the comments.  Please don't repeat the numerical values in the comments.

Reply:
Done